api/eds.proto

```diff
@@ -87,21 +90,24 @@ message LbEndpoint {
   // The limit of 128 is somewhat arbitrary, but is applied due to performance
   // concerns with the current implementation and can be removed when
   // https://github.com/envoyproxy/envoy/issues/1285 is fixed.
```

Comment:
There are a bunch of messages and URLs in the above two comment blocks. Can you add markup for them?